### PR TITLE
tests for templates

### PR DESCRIPTION
**Pull request by oleg08**
code review:
- [ ] @CHILL99 
- [ ] @krisberry 
- [ ] @natalya5 
- [x] @DimaSmagulov 
- [x] @OrestF 
- [ ] @rtershak 
- [x] @vvs79 

2nd lvl review:

- [ ] @Kmalynss
- [x] @serhiym 
- [ ] @ruba-ruba

### CODE REVIEW

File: test/fixtures/templates.yml [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
 one:
-  name: MyString
+  id: 1
+  name: MyName
+
+
 
 two:
+  id: 2
   name: MyString
```
Comment (krisberry) [inline test/fixtures/templates.yml]: You have to use FactoryGirl instead of fixtures.

Comment (serhiym) [inline test/fixtures/templates.yml]: :+1:  @krisberry

**Comment by serhiym** [comment]
@oleg08 - just change fixtures to factory - other looks good

**Comment by DimaSmagulov** [comment]
reviewed

**Comment by vvs79** [comment]
reviewed

File: test/fixtures/templates.yml (hunk 1 above)
Comment (Kmalynss) [inline test/fixtures/templates.yml]: @oleg08 please use FactoryGirl instead of fixtures as @krisberry said above.

**Comment by OrestF** [comment]
reviewed

**Comment by Kmalynss** [comment]
gtg

File: test/factories/templates.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :template do
+    id 1
```
Comment (Kmalynss) [inline test/factories/templates.rb]: id's are automatically incremented, please remove ```id 1```

**Pull request merged**